e2e/Wpf/HelloWorld.Bootstraper/App.xaml.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Windows;
+
+namespace HelloWorld
+{
+    /// <summary>
+    /// Interaction logic for App.xaml
+    /// </summary>
+    public partial class App : Application
+    {
+        protected override void OnStartup(StartupEventArgs e)
+        {
+            base.OnStartup(e);
+
+            var bs = new Bootstrapper();
+            bs.Run();
```

Comment:
you're right it is bs... haha

Reply:
that's my joke every time I write this line of code :)